RELEASE_PROCESS.md

```diff
@@ -21,27 +22,65 @@
+* Get someone to merge the PR before you go to the next step,
 
-* You can create a dummy commit in there, the only purpose is to make the CI running and start generating the images.
+* Create a PR in <https://github.com/openshift/tektoncd-pipeline> against `openshift/release-v${RELEASE}` for CI to pickup. The base branch for this PR will be against `openshift/tektoncd-pipeline:release-v${RELEASE}`. See here for an [example](https://github.com/openshift/tektoncd-pipeline/pull/26). You can run this script to automate all of it :
 
-* If there is a new binary generated for docker images then make sure you follow the section [New Images](#new-images) of this document
+    ```bash
+    USER_REMOTE="youruseremote"
+    git checkout -b test-release-v${RELEASE} release-v${RELEASE}
+    echo "$(date)" > ci
+    git add ci;git commit -m "CITest: v${RELEASE}"
+    git push ${USER_REMOTE} test-release-v${RELEASE}
+    echo "https://github.com/openshift/tektoncd-pipeline/compare/release-v${RELEASE}...${USER_REMOTE}:test-release-v${RELEASE}?expand=1"
+    ```
 
-* Make sure CI has picked up in your new PR and if that succedeed it means you have setup things successfully 🎉
+* After the CI tests passed, it will have the images generated and you can `/close/` the CITEST PR  🎉
 
 ### Generate release.yaml
 
 * Create a branch based on `openshift/release-v${RELEASE}` and run the command :
 
-`make generate-release RELEASE_VERSION=${RELEASE}`
+```bash
+    USER_REMOTE="youruseremote"
+    git checkout -b release-yaml-v${RELEASE} release-v${RELEASE}
+    make generate-release RELEASE_VERSION=${RELEASE}
+    git add openshift/release/tektoncd-pipeline-v${RELEASE}.yaml
+    git commit -m "Releasing release.yaml v${RELEASE}"
+    git push ${USER_REMOTE} release-yaml-v${RELEASE}
+    echo "https://github.com/openshift/tektoncd-pipeline/compare/release-v${RELEASE}...${USER_REMOTE}:release-yaml-v${RELEASE}?expand=1"
+```
+
+* This will generate a file in
+  `openshift/release/tektoncd-pipeline-${RELEASE}.yaml` commit and push it to
+  your git's USER_REMOTE and then you should have link to make your PR against it.
+
+* When you get it merged then you are good to go!
+
+### Other components
+
+### CLI
+
+* You need to make sure the upstream CLI dependency is updated to the new version.
+
+### Catalog
 
-This will generate a file in `openshift/release/tektoncd-pipeline-${RELEASE}.yaml` which you can add and create a PR for it (against the `openshift/tektoncd-pipeline` repository and `release-v${RELEASE}` branch)
+* Catalog needs to be tagged to the new version in downstream :
+
+    https://github.com/openshift/tektoncd-catalog
+
+    # TODO: To be filed by Vincent,
+
+* Images shipped with catalog needs to be tagged for quay mirroring
+
+    # TODO: TO be filled by Vincent
```

Comment:
Right now:
- Update to `:nightly` on the `s2i` tasks, and run tests
- In case of success, pull the `quay.io/openshift-pipeline/s2i` image, and tag it with release tag (e.g. `:v0.8.0`) and `:latest`, push back
- Update to the release tag on the `s2i` tasks

TODO: automate that part (script, ci/cd, …)

Reply:
I am not sure I understand fully 👼